### DISCUSSION TITLE
Kernels not upgrading beyond 6.9.3 in Ubuntu 24.04

### DISCUSSION THREAD

**Discussion by TheOriginalFreddoFrog** [bkw777/mainline#340] [General]
Hi Folks,

It is noted (at the time of writing) that Kernel 6.9.8 is the "current release". Mainline (at the time of writing) will only deliver Kernel Version 6.9.3  into Ubuntu 24.04.

Are there any issues with U 24.04 and or project itself leading to these outcomes?

Is there anything that we can as a community assist with?

Thanks

FF

**Comment by VK3SIR** [comment]
It is direct updating now to 6.9.9 as of 2024-07-12

Thanks Devs. Please sing out is assistance needed as inferred in FF's original post.

**Reply by simonepernice** [reply]
Today, as of 30th March 2025, running Ubuntu 24.04.2 LTS, I cannot download an updated kernel.

I get the following error:

mainline 1.4.12
install 6.12.21 
6.12.21 download failed
Install: no installable kernels specified
mainline: done

Is there a way to fix it?

Thank you,
Simone
